### PR TITLE
Feat(content): Remnant: Smew into the shipyard and fleets

### PR DESCRIPTION
**Pull request by Zitchas**
**Content (Artwork / Missions / Jobs)**

## Summary
This PR implements the recently merged Smew:
- Adds to the shipyard
- Adds a miner (with a thrasher), a support (with an inhibitor), and a runabout (with an hd) variants.
- Adds all three of these to Remnant fleets in a few instances.
- Including one Ibis+2 miners+4 terns variant for mining operations.

### CODE REVIEW

File: data/remnant/remnant ships.txt [hunk 1]
```diff
@@ -1192,6 +1192,17 @@ ship "Smew"
...
+	outfits
+		"Thrasher Cannon"
+
+ship "Smew" "Smew (Support)"
+	outfits
+		"Inhibitor Cannon"	
+
+ship "Smew" "Smew (Runabout)"
+	outfits
+		"Hyperdrive"
```
Comment (Amazinite) [inline data/remnant/remnant ships.txt:1205]: Remember that outfits aren't added. This is literally three ships equipped with only a thasher, inhibitor, or hyperdrive.

Comment (warp-core) [inline data/remnant/remnant ships.txt:1205]: Variants don't work like that. These will override the outfits of the base models, resulting in unusable ships.
We don't have an `add outfits` option at the moment, so you'll have to redefine the full outfit set.
It also doesn't have enough weapon capacity for an Inhibitor Cannon.

Comment (Zitchas) [reply data/remnant/remnant ships.txt]: lol. Removed the inhibitor variant, and expanded the outfit listing.

Comment (Zitchas) [reply data/remnant/remnant ships.txt]: Got it, and fixed. Thanks!

File: data/remnant/remnant ships.txt [hunk 2]
```diff
@@ -1192,6 +1192,25 @@ ship "Smew"
...
 
+ship "Smew" "Smew (Miner)"
+	outfits
+		"Millennium Cell"
+		"Quantum Key Stone"
+		"Thermoelectric Cooler"
+		
+		"Anvil-Class Engine"
+		
+		"Thrasher Cannon"
```
Comment (warp-core) [inline data/remnant/remnant ships.txt:1203]:
```suggestion
		"Thrasher Cannon"
		
		"Millennium Cell"
		"Quantum Key Stone"
		"Thermoelectric Cooler"
		
		"Anvil-Class Engine"
```

File: data/remnant/remnant ships.txt [hunk 3]
```diff
@@ -1192,6 +1192,25 @@ ship "Smew"
...
+
+ship "Smew" "Smew (Runabout)"
+	outfits
+		"Millennium Cell"
+		"Quantum Key Stone"
+		"Thermoelectric Cooler"
+		
+		"Anvil-Class Engine"
+
```
Comment (warp-core) [inline data/remnant/remnant ships.txt:1212]:
```suggestion
```

**Review by warp-core — COMMENTED** [review]
Per conventional ordering.

**Review by warp-core — APPROVED** [review]

**Pull request merged**